### ISSUE TITLE
gitlab MR -> github PR support

### ISSUE THREAD

**Issue by mash-graz** [zupzup/gitlab-push-and-mr#5]
it would be extremely useful, if your tool would be able to handle github PRs as well utilizing the same interface, because preferring gitlab for the actual private development work while contributing code to github hosted upstream repositories is a very common scenario, which isn't satisfyingly supported by any other tool right now.

**Comment by zupzup** [comment]
I agree that would be useful. However, this repo was really only planned as a code-repository for my blog-post about this.

Especially the name would probably have to change, if the tool were to support more backends than Gitlab. :D 

Anyway, if there is interest in such a tool, it might make sense to move it out here and make it more general. :)

**Comment by mash-graz** [comment]
yes -- it would be indeed a change/improvement, which goes far beyond the actual implementation, nevertheless it could be a really interesting project, because there is nothing similar available right now.
but it's of course up to you, whether you like to work any further on this kind of stuff at all. i just wanted to send you a little bit of feedback resp. new inspirations. ;)
